install_config/configuring_nsxtsdn.adoc

```diff
@@ -0,0 +1,217 @@
+nsx_api_managers='nsxmgr.example.com'
+nsx_api_user='nsx_admin'
+nsx_api_password='nsx_api_password_example'
+nsx_tier0_router='LR-Tier-0'
+nsx_overlay_transport_zone='TZ-Overlay'
+nsx_container_ip_block='pod-networking'
+nsx_no_snat_ip_block='pod-nonat'
+nsx_external_ip_pool='pod-external'
+nsx_top_fw_section='containers-top'
+nsx_bottom_fw_section='containers-bottom'
+nsx_ovs_uplink_port='ens224'
+nsx_cni_url='http://websrv.example.com/nsx-cni-buildversion.x86_64.rpm'
+nsx_ovs_url='http://websrv.example.com/openvswitch-buildversion.rhel75-1.x86_64.rpm'
+nsx_kmod_ovs_url='http://websrv.example.com/kmod-openvswitch-buildversion.rhel75-1.el7.x86_64.rpm'
+nsx_insecure_ssl=true
+# vSphere Cloud Provider
+#openshift_cloudprovider_kind=vsphere
+#openshift_cloudprovider_vsphere_username='administrator@example.com'
+#openshift_cloudprovider_vsphere_password='viadmin_password'
+#openshift_cloudprovider_vsphere_host='vcsa.example.com'
+#openshift_cloudprovider_vsphere_datacenter='Example-Datacenter'
+#openshift_cloudprovider_vsphere_cluster='example-Cluster'
+#openshift_cloudprovider_vsphere_resource_pool='ocp'
+#openshift_cloudprovider_vsphere_datastore='example-Datastore-name'
+#openshift_cloudprovider_vsphere_folder='ocp'
+[masters]
+master01.example.com
+master02.example.com
+master03.example.com
+[etcd]
+master01.example.com
+master02.example.com
+master03.example.com
+[nodes]
+master01.example.com ansible_ssh_host=192.168.220.2 OpenShift_node_group_name='node-config-master' OpenShift_ip=192.168.220.2
+master02.example.com ansible_ssh_host=192.168.220.3 OpenShift_node_group_name='node-config-master' OpenShift_ip=192.168.220.3
+master03.example.com ansible_ssh_host=192.168.220.4 OpenShift_node_group_name='node-config-master' OpenShift_ip=192.168.220.4
+node01.example.com ansible_ssh_host=192.168.220.5 OpenShift_node_group_name='node-config-infra' OpenShift_ip=192.168.220.5
+#node02.example.com ansible_ssh_host=192.168.220.6 OpenShift_node_group_name='node-config-infra' OpenShift_ip=192.168.220.6
+node03.example.com ansible_ssh_host=192.168.220.7 OpenShift_node_group_name='node-config-compute' OpenShift_ip=192.168.220.7
+node04.example.com ansible_ssh_host=192.168.220.8 OpenShift_node_group_name='node-config-compute' OpenShift_ip=192.168.220.8
+----
+
+.Procedure
+
+. Run the following command to install required software packages, if any, for {product-title}:
++
+----
+$ ansible-playbook -i hosts openshift-ansible/playbooks/prerequisites.yml
+----
+
+. Ensure that the NCP Container image is downloaded locally on all nodes
+
+. After the playbook has successfully executed, run the following command on all nodes:
++
+----
+$ docker load -i nsx-ncp-rhel-xxx.tar
+----
+
+. Get the image name and retag it:
++
+----
+$ docker images
+$ docker image tag registry.local/xxxxx/nsx-ncp-rhel nsx-ncp
+----
+
+. Deploy the {product-title} cluster:
+----
+$ ansible-playbook -i hosts openshift-ansible/playbooks/deploy_cluster.yml
```

Comment:
@vikram-redhat Do we want to refer people out to the OpenShift install docs here and remove the command?